bindings/matrix-sdk-ffi/src/room_alias.rs

```diff
@@ -0,0 +1,22 @@
+use regex::Regex;
+use ruma::RoomAliasId;
+
+/// Verifies the passed `String` matches the expected room alias format.
+#[matrix_sdk_ffi_macros::export]
+fn is_room_alias_format_valid(alias: String) -> bool {
+    RoomAliasId::parse(alias).is_ok()
+}
+
+/// Transforms a Room's display name into a valid room alias.
```

Comment:
I wonder if we should implement this on the `DisplayName` type, seem like it might be generally useful.